pages/index.html

```diff
@@ -5,8 +5,68 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet" />
+  <link href="styles/index.css" rel="stylesheet" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
 </head>
+
 <body>
+
+  <!-- <header>/ -->
+    <!-- <h1> Soleil - Software Developer </h1> -->
+  <!-- </header> -->
+  <div class="wrapper">
+  <nav class="logo">
+    <div class="meet-me">
+      <!-- <img src="images/Black Pink Bold Elegant Monogram Personal Brand Logo.png"> -->
+      <ul class="nav-area">
+        <li> <a href="index.html">Home</a></li>
+        <li> <a href="about.html">About</a></li>
+        <li> <a href="portfolio.html">Portfolio</a></li>
+        <li> <a href="/portfolio/">Services</a></li>
+      </ul>
+      </div>
+      
+  </nav>
+    <div>
+      <h1>Soleil Herring<br> Software Developer</h1>
+    </div>
```

Comment:
Headings can also stand alone on their own. Their default styling is basically a `div` already.